app/views/compare/_summary.html.erb

```diff
@@ -1,16 +1,46 @@
 <div class="p-3 mb-3 bg-light rounded border clearfix">
-  <%= link_to t('compare.filter.change_benchmark'), benchmarks_compare_index_path(index_params.except(:benchmark)), class: 'btn btn-sm float-right ml-2' if action_name == 'show' %>
+  <%= if action_name != 'benchmarks'
```

Comment:
The only thing that has changed here technically is as follows. All other stuff is erblint related! From
`action_name == 'show'
`
to:
`action_name != 'benchmarks'
`